statvar_imports/UNData/UNData_metadata.csv

```diff
@@ -0,0 +1,28 @@
+parameter,value
+url,"https://data.un.org/Handlers/DownloadHandler.ashx?DataFilter=tableCode:240&DataMartId=POP&Format=csv&c=2,3,6,8,10,12,14,16,17,18&s=_countryEnglishNameOrderBy:asc,refYear:desc,areaCode:asc"
+description,"UNData City population by sex, city and city type"
+#place_type,UNData
+#places_within,
+start_date,1972
+end_date,2024
+release_frequency,1 Year
+process,
+comments,
+output_columns,"observationAbout,observationDate,value,variableMeasured,unit,scalingFactor"
+#places_resolved_csv,https://docs.google.com/spreadsheets/d/1VsmXyn35cSIEMtoOOf5IKeJL2j0T2qFD-uWWBQTZIYQ/edit?gid=114243120#gid=114243120
+header_rows,1
+mapped_rows,1
+mapped_columns,11
+#word_delimiter,
+#drop_statvars_without_svobs,1
+debug,0
+#input_rows,5
+#places_resolved_csv,https://docs.google.com/spreadsheets/d/1U399H_LdQMQ65PiWsc7sJ-50AbTmm3LAl8TuhtdYabY/edit?resourcekey=0-LZxQZaPr1YHFlyJJXsU5KA&gid=499539209#gid=499539209
```

Comment:
pls add the places_resolved csv to this folder too. I assume the input doesn;t have to be modified anymore and the observationAbout can be set directly to the place name and pv_map can map place name to dcid.